documentation/v0_2_0/dev/guide.md

```diff
@@ -148,3 +148,27 @@ machinectl shell -q flotta@.host /usr/bin/journalctl --user
 - *Go/Lint*: Checks that code is correctly linted
 - *Go/SecurityScanner*: Checks that the current code is statically checked by
   [gosec](https://github.com/securego/gosec)
+
+# Developer CLI
+
+### Prerequisites
+- Install the flotta operator as mentioned [here](https://project-flotta.io/documentation/v0_2_0/dev/installation.html).
```

Comment:
What do you think about using `https://project-flotta.io/documentation/latest/dev/installation.html`?

Reply:
Done :)